app/services/etherdelta_observer.py

```diff
@@ -97,6 +97,7 @@
         "updated"  = $2
     WHERE "signature" = $3
 """ # Totally a duplicate of contract event recorder SQL
+from ..tasks.update_order import update_order_by_signature
```

Comment:
Are the changes to this file related to Huey or unrelated?

Reply:
Yeap, this imports the function that will enqueue a background job (used below)

Reply:
I don't see where that function is defined in these changes?

Reply:
Over in [`app/tasks/update_order.py`](https://github.com/forkdelta/backend-replacement/pull/20/files#diff-862b5a8b0e0a329ee38e180d95676f1d)

Reply:
Weird, that wasn't showing up yet. Looks good.